client/src/App.js

```diff
@@ -3,8 +3,12 @@ import Login from "./containers/login/Login"
 import Dashboard from "./containers/dashboard/Dashboard"
  import {Route, Switch,Redirect} from 'react-router-dom'
 import { connect } from "react-redux";
+import EditHours from "./containers/editHours/EditHours"
+import PopUpEdit from "./containers/PopUp/PopUpEdit"
 /* import io from 'socket.io-client'
```

Comment:
In a general sense when pushing into a branch like `master` or `develop` it's recommended to remove these comments :) Just to keep the repo nice and clean

Reply:
After I figure out how to do the login with google I will clean everything.

Reply:
A suggestion could be that you create a separate branch where you do the Google Auth testing and integrate it once it's finished. Then this one wouldn't need to the comments